spec/requests/posts_request_spec.rb

```diff
@@ -1,17 +1,16 @@
 require 'rails_helper'
 
-RSpec.describe 'Posts controller' do
+RSpec.describe 'Testing Posts controller' do
   it 'should return a post' do
-    get '/users/1/posts/'
+    user = User.create!(id: 1, name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher fr Mexico.',
+                        posts_counter: 0)
+    Post.create(id: 1, author: user, title: 'Hello', text: 'This is my first post', comments_counter: 0,
+                likes_counter: 0)
+    get '/user/1/posts'
     expect(response).to have_http_status(:success)
     expect(response.status).to eq(200)
-    expect(response.body).to include('Post 1')
-    expect(response.body).to include('Lorem ipsum dolor sit amet, consectetur')
-  end
-  it 'should return single post text' do
-    get '/users/1/posts/1'
-    expect(response).to have_http_status(:success)
-    expect(response.status).to eq(200)
-    expect(response.body).to include('Lorem ipsum dolor sit amet, consectetur adipiscing elit,')
+    expect(response.body).to include('Hello')
+    expect(response.body).to include('This is my first post')
+    expect(response).to render_template('shared/_navbar', 'posts/index')
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=to%20your%20project.-,Create%20a%20Request%20spec%20file%20for%20all%20your%20controllers.,-Make%20sure%20that), You need to test both index and show for each controller, So Kindly add a request tests to the `show` path as you did with `index` path. and also make sure that the test type is `request`.